tc/sgx/trusted_worker_manager/enclave/CMakeLists.txt

```diff
@@ -171,6 +171,11 @@ ELSE()
   ADD_DEFINITIONS(-DTCF_DEBUG_BUILD=0)
 ENDIF()
 
+# Enable enclave logs
+IF(DEFINED ENV{TCF_ENCLAVE_LOG} AND "$ENV{TCF_ENCLAVE_LOG}" STREQUAL "1")
+  ADD_DEFINITIONS(-DTCF_ENCLAVE_LOG=1)
```

Comment:
From security perspective no logs are allowed in the release build so we need to modify our definitions to have 
TCF_SGX_SYM - to build TCF in SGX simulation mode
TCF_DEBUG_BUILD - to build  for either HW or simulation mode
TCF_ENCLAVE_LOG - allows logging, but only if TCF_DEBUG_BUILD is defined, e.g.
IF(DEFINED ENV{TCF_ENCLAVE_LOG} AND "$ENV{TCF_ENCLAVE_LOG}" STREQUAL "1" AND TCF_DEBUG_BUILD} AND "$ENV{TCF_DEBUG_BUILD}" STREQUAL "1")

We must have a single clearly defined macro that turns off all debug support presenting a potential security issues

Reply:
Currently TCF_DEBUG_BUILD is used to enable debugging symbols (-g option in CMakeList.txt). But I was thinking logging (via ocall) is independent of enabling debugging symbols. So you are suggesting to enable logs we have to have debug symbols ?